src/openms/include/OpenMS/CHEMISTRY/EmpiricalFormula.h

```diff
@@ -126,6 +127,21 @@ namespace OpenMS
     double getAverageWeight() const;
 
     /**
+      @brief Fills this EmpiricalFormula with an approximate elemental composition for a given average weight and approximate elemental stoichiometry
+
+      @return integer return flag describing result of the approximation. 0 = no problems, 1 = negative hydrogens requested.
```

Comment:
I agree with Chris - if there are only two outcomes, please return a bool, with "true" for "success".